app/services/bulk_upload/lettings/year2023/row_parser.rb

```diff
@@ -435,7 +435,13 @@ def valid?
       fields = field_mapping_for_errors[error.attribute] || []
 
       fields.each do |field|
-        unless errors.include?(field)
+        next if errors.include?(field)
+
+        question = log.form.get_question(error.attribute, log)
+
+        if question.present? && setup_question?(question)
+          errors.add(field, error.message, category: :setup)
+        else
```

Comment:
Much nicer! Can sales have the same?

Reply:
Ah, yes, sorry, totally missed that.. Thanks!